### PR TITLE
New GDPR popup

### PR DESCRIPTION
**Pull request by Micpol**
Changed layout for both portrait and landscape GDPR popup.

### CODE REVIEW

**Review by michal-bogucki — COMMENTED** [review]
michal-bogucki do raportu

File: library/src/main/java/pl/netigen/rodo/GDPRDialogFragment.kt [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        this.isNoAdsAvailable = isNoAdsAvailable
+    }
+
+    fun isNetworkOn(): Boolean {
+        val context = context
+        val cm: ConnectivityManager?
+        if (context == null) {
+            return false
+        }
+        cm = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
```
Comment (WrobelKrakow) [inline library/src/main/java/pl/netigen/rodo/GDPRDialogFragment.kt]: cm = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager - here can be thrown TypeCastException

should be:
val connectivityManager = getSystemService(Context.CONNECTIVITY_SERVICE)
      if (connectivityManager is ConnectivityManager) {

**Review by WrobelKrakow — CHANGES_REQUESTED** [review]

File: library/src/main/java/pl/netigen/rodo/GDPRDialogFragment.kt (hunk 1 above)
Comment (Micpol) [reply library/src/main/java/pl/netigen/rodo/GDPRDialogFragment.kt]: Change done it now is

```Kotlin
val connectivityManager = context?.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager?
                ?: return false
```

P.S. Try using MARKDOWN so comments with code will be more readable

File: library/src/main/java/pl/netigen/netigenapi/BaseSplashActivity.java [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package pl.netigen.netigenapi;
 
-
 import android.os.Bundle;
 import android.os.Handler;
+
```
Comment (marcin03) [inline library/src/main/java/pl/netigen/netigenapi/BaseSplashActivity.java:5]: A po co to komu?

**Review by marcin03 — APPROVED** [review]
I had a lot of fun

File: library/src/main/java/pl/netigen/netigenapi/BaseSplashActivity.java (hunk 2 above)
Comment (Micpol) [reply library/src/main/java/pl/netigen/netigenapi/BaseSplashActivity.java:5]: Ops

**Review by WrobelKrakow — APPROVED** [review]

**Pull request merged**